src/seedu/addressbook/data/person/UniquePersonList.java

```diff
@@ -122,6 +122,10 @@ public void remove(ReadOnlyPerson toRemove) throws PersonNotFoundException {
         }
     }
 
+    public void sort() {
+        this.internalList.sort(Person.nameComparator);
```

Comment:
`this` not necessary here (Check the coding standards under "Avoid unnecessary use of this with fields.")

Reply:
Fixed. Modified the user guide and added the javadocs too (see latest commit), missed that it was required.